app/models/custom_field.rb

```diff
@@ -32,9 +32,8 @@ class CustomField < ActiveRecord::Base
 
   has_many :custom_values, dependent: :delete_all
   acts_as_list scope: 'type = \'#{self.class}\''
-  translates :name,
-             :default_value,
-             :possible_values
+  translates :name, fallbacks_for_empty_translations: false
```

Comment:
@ulferts with `fallbacks_for_empty_translations: true`, translations will fallback when `nil` or an empty string. with `fallbacks_for_empty_translations: false` (default), only when the `nil`. I'm not sure why I specified a default option here: I think this was the remnant of some me playing around with different options.